Makefile

```diff
@@ -34,7 +34,7 @@ vet: sdk-generate
 	$(Q)go vet ./...
 
 test: vet format
-	$(Q)go test ./...
+	$(Q)GOCACHE=off go test ./...
```

Comment:
Is this to force-run tests even when there haven't been code changes? I noticed I wasn't getting tests by just changing yaml files.

Reply:
correct